### PR TITLE
Introduce FakeSSLTools

### PR DESCRIPTION
**Pull request by dwijnand**
Taking bits of play.it.test.ServerEndpoint.SelfSigned.

Also:
* Make chunks of FakeKeyStore back into an object
* Fix wget call in validate-pom-files.sh

### CODE REVIEW

File: ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2015 - 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.typesafe.sslconfig.ssl
```
Comment (marcospereira) [inline ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala]: Should we move it to a `testkit` package to make it clear that it should not be used in production code?

Comment (dwijnand) [reply ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala]: Maybe one day, for now this is kind of like `FakeKeyStore`. Not really for production, but not dangerous like a `lazy val` is to have here.

Comment (renatocaval) [reply ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala]: I don't get your point about `lazy val`.

Comment (dwijnand) [reply ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/FakeSSLTools.scala]: @renatocaval not as dangerous as https://github.com/lightbend/ssl-config/pull/119#discussion_r221280971

**Pull request merged**